src/MicrosoftTeams.ts

```diff
@@ -282,6 +284,42 @@ namespace microsoftTeams {
         }
     }
 
+    /**
+     * Registers a handler for user presses of the Team client's back button. Experiences that maintain an internal
+     * navigation stack should use this handler to navigate the user back within their frame. If an app finds
+     * that after running its back button handler it cannot handle the event it should call the navigateBack
+     * method to ask the Teams client to handle it instead.
+     * @param handler The handler to invoke when the user presses their Team client's back button.
+     */
+    export function registerBackButtonHandler(handler: () => void): void {
+        ensureInitialized();
+
+        backButtonPressHandler = handler;
+    }
+
+    function handleBackButtonPress(): void {
+        if (backButtonPressHandler) {
+            backButtonPressHandler();
+         } else {
+             navigateBack();
+         }
+    }
+
+    /**
+     * Navigates back in the Teams client. See registerBackButtonHandler for more information on when
+     * it's appropriate to use this method.
+     */
+    export function navigateBack(): void {
```

Comment:
This will always mean close the opened tab right?

Reply:
Yes

Reply:
Yup